O/OpenBLAS/common.jl

```diff
@@ -84,7 +84,7 @@ function openblas_script(;num_64bit_threads::Integer=32, openblas32::Bool=false,
 
     # On Intel architectures, engage DYNAMIC_ARCH
     if [[ ${proc_family} == intel ]]; then
-        flags+=(TARGET= DYNAMIC_ARCH=1)
+        flags+=(TARGET=NEHALEM DYNAMIC_ARCH=1)
```

Comment:
Will this work with the generic x86_64 microarchitecture?

Reply:
GENERIC is now being specified.